New practice item

moudgilz opened the pull request.

Commit:
new pages added

Commit:
Merge branch 'master' of https://github.com/jagdishkumar17/react-js-webpack

# Conflicts:
#	src/index.js

Commit:
newPracticeItem added

Commit:
default page set again

moudgilz commented:
here in this branch some practice folder added,
beside index.js file updated as per requirements, so any time any file required to run just simply un comment that link and run

Commit:
unused link removed

jagdishkumar17 commented on src/app/components/Events/events.jsx:
```diff
@@ -0,0 +1,52 @@
+      return (
+         <div>
+            <ChildComponent myPropOne={this.state.prop1} myPropTwo={this.updateStatus}
+               propEvent={this.buttonEventCheck} proEventValue={this.state.prop2} />
```
can you please add example for how we can pass parameter to buttonEventCheck function on button click

moudgilz replied:
ok

moudgilz replied:
i have taken here onChangeEvent instead onClick

moudgilz replied:
button click event with parameter has called up

Commit:
here event is modified,
now once you type anything in textbox of child page the value will pass back to parent form using event

moudgilz reviewed (commented):
here event is modified,
now once you type anything in textbox of child page the value will pass back to parent form using event

Commit:
comment added

Commit:
button click event with parameter has called up

Commit:
commented code removed

Commit:
code added

Commit:
Merge branch 'master' of https://github.com/jagdishkumar17/react-js-webpack

# Conflicts:
#	src/index.js

Commit:
changes added

Commit:
Merge branch 'master' into newPracticeItem

# Conflicts:
#	src/index.js
